compiler/AST/TryStmt.cpp

```diff
@@ -20,6 +20,15 @@
 #include "TryStmt.h"
 #include "AstVisitor.h"
  
+BlockStmt* TryStmt::build(bool tryBang, BlockStmt* body) {
```

Comment:
Trivial observation.  I am pretty sure this could be just

BlockStmt* TryStmt::build(bool tryBang, BlockStmt* body) {
  return new BlockStmt(new TryStmt(tryBang, body));
}